### DISCUSSION TITLE
pp rework

### DISCUSSION THREAD

**Discussion by wehatekanji** [acrylic-style/ezpp#6] [General]
just wanted to know if you were planning on updating this since its extremely helpful! :D if not thanks anyways ^^

**Comment by acrylic-style** [comment]
I don't know if i'm going to update it or not. It just takes a ton of time but I don't have much time to spend on it.
At least I can spend some time on it, so i think i'll try to write code later.

**Comment by wehatekanji** [comment]
Damn didn’t know it took a lot of time haha if its to much work do not! Feel obligated to do it ^:^ the mod is only like 5-10% off so its not that bad!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
